eclair-core/src/main/scala/fr/acinq/eclair/wire/LightningMessageTypes.scala

```diff
@@ -52,7 +52,8 @@ case class OpenChannel(chainHash: BinaryData,
                        paymentBasepoint: Point,
                        delayedPaymentBasepoint: Point,
                        firstPerCommitmentPoint: Point,
-                       channelFlags: Byte) extends ChannelMessage with HasTemporaryChannelId
+                       channelFlags: Byte,
+                       htlcBasepoint: Point) extends ChannelMessage with HasTemporaryChannelId
```

Comment:
field order has been changed, in the specs it is now:

   * [`33`:`payment_basepoint`]
    * [`33`:`delayed_payment_basepoint`]
    * [`33`:`htlc_basepoint`]
    * [`33`:`first_per_commitment_point`]
    * [`1`:`channel_flags`]